documentation/src/main/docs/faq.adoc

```diff
@@ -23,6 +30,17 @@ If you need to generate a sequence of items use, `producePublisher`:
 include::../../../src/test/java/snippets/HowToTransformTest.java[tags=multi]
 ----
 
+[NOTE]
+.Unchecked
+====
+Mutiny provides a set of utility classes to handle functions / consumers / suppliers that throw _checked_ exceptions.
+`io.smallrye.mutiny.unchecked.Unchecked` let you wrap code throwing checked exceptions into the expected Java type.
```

Comment:
"lets you wrap the code"?

Reply:
is: "wrap code throwing checked exceptions into Java functions/suppliers/consumers" ?

Reply:
s/wrap/wraps, but that works too